File: task_02.mjs
import { readFileSync } from 'fs';

if (!process.argv[2]) {
  console.error('usage: task_02.mjs <ext>');
  process.exit(1);
}

try {
  const count = readFileSync('random.txt', 'utf8')
    .split('\n')
    .filter(l => l === process.argv[2]).length;
  console.log(`${process.argv[2]}: ${count}`);
} catch (e) {
  console.error(`✗ random.txt puudub`);
  process.exit(1);
}

File: task_02.py
import sys

if len(sys.argv) < 2:
    print("Kasutamine: task_02.py <laiend>", file=sys.stderr)
    sys.exit(1)

target = sys.argv[1]

try:
    with open("random.txt", encoding="utf-8") as f:
        count = sum(1 for line in f if line.strip() == target)
    print(f"{target}: {count}")
except FileNotFoundError:
    print("✗ random.txt puudub", file=sys.stderr)
    sys.exit(1)